### PR TITLE
Drop createrepo_c build deps

### PR DESCRIPTION
**Pull request by dralley**
We use the prebuilt wheels nowadays

[noissue]

### CODE REVIEW

**Comment by dralley** [comment]
Oh, right, we can't do this until we upgrade to Python 3.9, because createrepo_c doesn't publish wheels for 3.8

File: images/Containerfile.core.base [hunk 1]
```diff
@@ -45,15 +39,7 @@ RUN dnf -y install python3 python3-cryptography python3-devel && \
     dnf -y install python3-libcomps && \
     dnf -y install libpq-devel && \
```
Comment (dralley) [inline images/Containerfile.core.base:48]: Do we remember what this is required for?  And openldap-devel?

**Comment by dralley** [comment]
@mikedep333 I think this is failing because we're building aarch64 images now, but the binary packages don't exist for createrepo_c, solv, and maybe libcomps, so it falls back to trying to build from source.

I guess the only alternatives are qemu, or waiting until GHA has ARM runners available.

File: images/Containerfile.core.base (hunk 1 above)
Comment (dralley) [reply images/Containerfile.core.base:48]: @mikedep333 @mdellweg

**Comment by decko** [comment]
> @mikedep333 I think this is failing because we're building aarch64 images now, but the binary packages don't exist for createrepo_c, solv, and maybe libcomps, so it falls back to trying to build from source.
> 
> I guess the only alternatives are qemu, or waiting until GHA has ARM runners available.

And look what we got https://github.blog/2024-06-03-arm64-on-github-actions-powering-faster-more-efficient-build-systems/

**Comment by dralley** [comment]
But not on free plans yet.  And older branches use older versions of createrepo_c and libcomps and such, for which no binary arm packages are uploaded, so we might have to wait a bit for some of those to phase out.

File: images/Containerfile.core.base [hunk 2]
```diff
@@ -53,15 +47,7 @@ RUN dnf -y install python${PYTHON_VERSION} python3-cryptography python${PYTHON_V
...
-    dnf -y install swig && \
-    dnf -y install podman fuse-overlayfs buildah --exclude container-selinux && \
-    dnf -y install xz && \
-    dnf -y install libmodulemd-devel && \
-    dnf -y install libcomps-devel && \
-    dnf -y install zchunk-devel && \
-    dnf -y install ninja-build && \
-    dnf -y install cairo-devel cmake gobject-introspection-devel cairo-gobject-devel && \
-    dnf -y install libcurl-devel libxml2-devel sqlite-devel file-devel && \
+-   dnf -y install podman fuse-overlayfs buildah --exclude container-selinux && \
```
Comment (gerrod3) [inline images/Containerfile.core.base]: You have a dash here on line 50 that needs to be removed.

**Review by gerrod3 — APPROVED** [review]
Nice!

**Pull request merged**